configs/watchtower-template.config.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
```

Comment:
Would be good to add some notes on what the config options do. Either here or in the readme

Reply:
Updated with docs for context. I suggest they run `keg watchtower start -h` to learn what the options do, that way we're not duplicating the documentation anywhere.

Reply:
works for me